README.md

```diff
@@ -50,27 +50,68 @@ Initialize Intercom in your `AppDelegate.m`
 
 More instructions here: [Intercom for Android](https://github.com/intercom/intercom-android)
 
-```
-Intercom.initialize(getApplicationContext(), "your api key", "your app id");
+Your Android Application should look like:
+
+```java
+// ...
+import com.robinpowered.react.Intercom.IntercomPackage;
+import io.intercom.android.sdk.Intercom;
+
+public class MainApplication extends Application {
+
+  @Override
+  public void onCreate() {
+    super.onCreate();
+    Intercom.initialize(this, "your api key", "your app id");
+    // ...
+  }
+  
+  public List<ReactPackage> getPackages() {
+    return Arrays.<ReactPackage>asList(
+        // ...
+        new IntercomPackage()
+	// ...
+    );
+  }
+}
+
 ```
 
-And in your *AndroidManifest.xml* file add the following lines within the `<application> ... </application>` tags
+And in your *AndroidManifest.xml* should look like
 ```xml
-<service
-     android:name="com.robinpowered.react.Intercom.IntercomIntentService"
-     android:exported="false">
-     <intent-filter
-        android:priority="999">
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.myapp"
+	  ...
+          xmlns:tools="http://schemas.android.com/tools"
```

Comment:
I would add comment  like `<!-- Add this line -->`

Reply:
it wouldn't be valid xml comment imo so I prefered to put "..."

You want something like that?

```xml
<manifest package="com.myapp"
          <!-- add this line -->
          xmlns:tools="http://schemas.android.com/tools"
    >

   <!-- comment -->
</manifest>
```

Reply:
What ever works the best :D Point is that there is comment pointing what you need to add

Reply:
this could be fine like this:

![image](https://user-images.githubusercontent.com/749374/33206959-456ec5c4-d10c-11e7-8361-ea696c924a89.png)